app/constants/colors.js

```diff
@@ -61,6 +63,26 @@ const colors = {
 
   MONO_DARK: '#202020',
   MONO_SECONDARY: '#757677',
+
+  background: '#EFEFEF',
+  errorBackground: 'red',
+  errorText: '#fff',
+  green: '#7ED321',
+  mainBlue: '#0161F2',
+  lightGray: '#D8D8D8',
+  lightBlue: '#EFF4F9',
+  buttonLightText: '#6B6B6B',
+  noticeBackground: tintColor,
+  noticeText: '#fff',
+  orange: '#FEB313',
+  pink: '#F1485B',
+  purple: '#525BD9',
+  tabBar: '#fefefe',
+  tabIconDefault: '#ccc',
+  tabIconSelected: tintColor,
+  tintColor,
+  warningBackground: '#EAEB5E',
+  warningText: '#666804',
```

Comment:
Let's be consistent with the naming convention.
Could you change those names to uppercase?

Reply:
also, Could we named as the color name not base on what we use the color?
example:
```js
warningText: '#666804'
```
to 
```js
YUKON_GOLD: '#666804'
```

http://chir.ag/projects/name-that-color/